guides/v2.3/magento-functional-testing-framework/2.3/commands/codeception.md

```diff
@@ -13,17 +13,13 @@ _This topic was updated due to the {{page.mftf-release}} MFTF release._
 
 {:.bs-callout .bs-callout-warning}
 We do not recommend using Codeception commands directly as they can break the MFTF basic workflow.
-All the Codeception commands you need are wrapped using the `mftf` tool.
+All the Codeception commands you need are wrapped using the [`mftf` tool][].
 
-See the [list of `mftf` commands](mftf.html). If you'd like to run the Codeception tests directly, run:
+To run Codecept, change your directory to `dev/tests/acceptance`.
```

Comment:
To run Codecept `directly`

Reply:
Per @aljcalandra , it is the only working way to run `codecept` commands in the MFTF without touching configuration.
Isn't it?
When I run `codecept` from `magento2` it requires bootstrapping and additional configuration. Which `codecept` should be used in which case: `magento2/vendor/bin/codecept` vs `magento2/dev/tests/acceptance/vendor/bin/codecept`?

Reply:
Definitely the `magento2/dev/tests/acceptance/vendor/bin/codecept`. I was only commenting that the word `directly` should be added to differentiate between running tests with `bin/mftf` (which uses codeception) and running Codecept by itself.

Reply:
Got it